css/components/navigation.css

```diff
@@ -0,0 +1,57 @@
+/* === css for navigation ==== */
+
+.navigation-container{
+    display: flex;
+    width: 100%;
+    align-items: center;
+    padding: 1.5rem .7rem;
+    justify-content: space-between;
+    background-color: #333;
+}
+.left h5{
+    font-size: var(--normal-font-size);
+    color: #fff;
+}
```

Comment:
Avoid fixed color values.

Reply:
sure will create a variable in color palette.

Reply:
Yeah that would do it.